vmwareengine/cloud-client/test_cluster.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import mock
+
+from google.cloud import vmwareengine_v1
+
+from create_cluster import create_cluster
+from create_custom_cluster import create_custom_cluster
+from delete_cluster import delete_cluster
+from update_cluster import update_cluster
+
+
+@mock.patch("google.cloud.vmwareengine_v1.VmwareEngineClient")
+def test_create_cluster(mock_client_class):
+    mock_client = mock_client_class.return_value
+
+    create_cluster("project11", "zone43", "my-cloud", "my-cluuuster", 99)
+
+    mock_client.create_cluster.assert_called_once()
+    print("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", list(mock_client.create_cluster.call_args[0]))
+    assert len(mock_client.create_cluster.call_args[0]) == 1
+    assert len(mock_client.create_cluster.call_args[1]) == 0
+    request = mock_client.create_cluster.call_args[0][0]
+    assert isinstance(request, vmwareengine_v1.CreateClusterRequest)
+    assert (
+        request.parent == "projects/project11/locations/zone43/privateClouds/my-cloud"
+    )
+    assert request.cluster.name == "my-cluuuster"
+    assert request.cluster.node_type_configs["standard-72"].node_count == 99
+
+
+@mock.patch("google.cloud.vmwareengine_v1.VmwareEngineClient")
+def test_create_custom_cluster(mock_client_class):
```

Comment:
Please consider adding tests for raising exceptions if number of nodes is below 3 in `create_cluster` and `create_custom_cluster`.

Reply:
Added.